CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.10.6] - 2023-02-16
+
+### Features
```

Comment:
The changelog is autogenerated from commit descriptions. Just make your commit message `feat: xyz` and I'll generate the changelog when cutting a release

Reply:
Done